qiskit/algorithms/amplitude_amplifiers/grover.py

```diff
@@ -111,11 +111,13 @@ def __init__(
         Args:
             iterations: Specify the number of iterations/power of Grover's operator to be checked.
                 * If an int, only one circuit is run with that power of the Grover operator.
-                If the number of solutions is known, this is option should be used with the optimal
+                If the number of solutions is known, this option should be used with the optimal
                 power. The optimal power can be computed with ``Grover.optimal_num_iterations``.
                 * If a list, all the powers in the list are run in the specified order.
                 * If an iterator, the powers yielded by the iterator are checked, until a maximum
                 number of iterations or maximum power is reached.
+                * If ``None``, the :obj:`.AmplificationProblem` provided must have an is_good_state, and
```

Comment:
is this intended or a typo?
```suggestion
                * If ``None``, the :obj:`AmplificationProblem` provided must have an is_good_state, and
```